tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/LinagoraCalendarEventAttendanceGetMethodContract.scala

```diff
@@ -317,7 +318,6 @@ trait LinagoraCalendarEventAttendanceGetMethodContract {
            |]""".stripMargin)
   }
 
-  @Disabled("A system session is used to fetch message flags from user inbox which does not seem to handle user mailbox isolation.")
   @Test
```

Comment:
Please add one test

```
GIVEN Alice delegated her INBOX to BOB
AND Alice has an ICS in her INBOX
WHEN bob do EventAttendance/get
THEN XXXX
```

(We need to document the current behaviour...

Reply:
It succeeds https://github.com/linagora/tmail-backend/pull/1447/commits/8e97b5a7842e4c85f2e488af560b9c068a864f55